lightning/src/ln/chan_utils.rs

```diff
@@ -38,6 +38,7 @@ use util::transaction_utils::sort_outputs;
 use ln::channel::INITIAL_COMMITMENT_NUMBER;
 use std::io::Read;
 use chain;
+use bitcoin::hashes::core::ops::Deref;
```

Comment:
The ordering is inconsistent in this file, but perhaps move this near related imports.

Reply:
And it's the wrong import to boot.  Fixing.